runtime/src/lib.rs

```diff
@@ -625,7 +710,12 @@ construct_runtime! {
 		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>},
 		Recovery: pallet_recovery::{Pallet, Call, Storage, Event<T>},
 		ParachainInfo: parachain_info::{Pallet, Storage, Config},
-		XcmHandler: cumulus_pallet_xcm_handler::{Pallet, Call, Event<T>, Origin},
+
+		// XCM helpers.
+		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>},
+		PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin},
+		CumulusXcm: cumulus_pallet_xcm::{Pallet, Origin},
+		Spambot: cumulus_ping::{Pallet, Call, Storage, Event<T>} = 99,
```

Comment:
why just assign this module as value 99?

Reply:
Copied from cumulus repo. Need to check

Reply:
I think `99` can be safely removed. Since the `construct_time` will be expanded as an `enum` along with other functions